files/graphdb_backup

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# This script utilizes the cloud backup functionality of GraphDB.
+# It is intended to be run in clustered topology.
+
+set -euo pipefail
+
+az login --identity
+
+RESOURCE_GROUP="$(curl -s -H Metadata:true "http://169.254.169.254/metadata/instance/compute/resourceGroupName?api-version=2021-01-01&format=text")"
```

Comment:
I don't like these variables here. I could use the image with another user data script when creating VMs, we should not restrict this, IMO. Also, the way how the script assumes that the first storage account and app config are the ones to be used... it's wrong as it is not portable to most subscriptions. Better to pass the variables as arguments.

Reply:
done